examples/raster.py

```diff
@@ -52,14 +52,15 @@ def update_image():
 dims.on_change('data', on_dims_change)
 dims_jscode = """
 var update_dims = function () {
-    var new_data = {};
-    new_data['height'] = [plot.get('frame').get('height')];
-    new_data['width'] = [plot.get('frame').get('width')];
-    new_data['xmin'] = [plot.get('x_range').get('start')];
-    new_data['ymin'] = [plot.get('y_range').get('start')];
-    new_data['xmax'] = [plot.get('x_range').get('end')];
-    new_data['ymax'] = [plot.get('y_range').get('end')];
-    dims.set('data', new_data);
+    var new_data = {
+        height: [plot.frame.height],
+        width: [plot.frame.width],
+        xmin: [plot.x_range.start],
+        ymin: [plot.y_range.start],
+        xmax: [plot.x_range.end],
+        ymax: [plot.y_range.end]
+    };
+    dims.data = new_data;
 };
 
 if (typeof throttle != 'undefined' && throttle != null) {
```

Comment:
On an unrelated note, this code is redundant, as `throttle != null` covers both cases, because `undefined` and `null` the same under implicit comparison. This `if` statement could simply read `if (throttle != null)`. This is btw. how `?` operator in coffeescript is implemented. Otherwise, one should use explicit comparison operators. In general one should use the later when writing pure javascript to avoid weird implicit conversions.